### DISCUSSION TITLE
what is this

### DISCUSSION THREAD

**Discussion by ADAGAMLUCA** [scripthookvdotnet/scripthookvdotnet#1432] [Q&A]
Unhandled exception in script "Main"!
NullReferenceException em
Main.OnKeyDown(Object sender, KeyEventArgs e)

When I press any arrow on my keyboard this appears

**Comment by kagikn** [comment]
The log besically says some of the method is trying to access some method or property via null reference, which doesn't make sense. Is that exception thrown from a script you coded? Report the log to the author(s) if you didn't code the script.
